src/assets/stylesheets/index.scss

```diff
@@ -1255,7 +1255,9 @@ section:focus >,
     box-shadow: none;
   }
   .navbar-brand-logo {
+    max-height: 100%;
     max-width: 155px;
+    min-height: 45px;
   }
```

Comment:
FYI: I think we should also specify a `max-height`. If one drops a thin logo such as
<img width="268" alt="Screenshot 2024-08-28 at 09 22 49" src="https://github.com/user-attachments/assets/de07fde1-972b-47f6-8e45-a99685a72dfa">

the navigation looks strange:
<img width="1570" alt="Screenshot 2024-08-28 at 09 26 30" src="https://github.com/user-attachments/assets/30ac64d7-4c0e-4107-9834-899c4dc02120">